mllib-dal/build.sh

```diff
@@ -11,9 +11,14 @@ if [[ -z $(which mvn) ]]; then
  exit 1
 fi
 
-if [[ -z $DAALROOT ]]; then
```

Comment:
You should use this block. 
-z checks if the varible exists and -e check if the path exists. they are both needed.